### DISCUSSION TITLE
How do I mark rows in the table non reactive?

### DISCUSSION THREAD

**Discussion by vikaskedia** [CodeDredd/pinia-orm#762] [Q&A]
I have a table with 100 rows and once I fetch them from the DB through an API they are never going to change.

Can I mark these rows as non reactive to speed up the app?

**Comment by CodeDredd** [comment]
Hmm don't know if i get it right. Maybe you can provide an example for me please ? 🙏 
Maybe doing this with piniaOptions in the model does the trick: https://github.com/vuejs/pinia/discussions/1220
But i think you have to adept the save action also.

**Comment by roy-rogerroger** [comment]
Could you provide an example? I would like to use this too, but can't figure out how to do use markRaw on certain fields or on the full data object.
